app/view_objects/results_view.rb

```diff
@@ -127,9 +127,10 @@ def sort_options
 
   def courses
     @courses ||= begin
-                   base_query = Course
-                     .includes(:provider).includes(:sites)
-                     .where(recruitment_cycle_year: Settings.current_cycle)
+                   base_query = Course.
+                     includes(site_statuses: [:site]).
+                     includes(provider: [:sites]).
```

Comment:
Do we need to add the sites to the provider? Or is this some weird JSONAPIClient thing where if we include sites for `site_statuses` we also have to include them for `provider`?

Reply:
👀

Reply:
Nah we don't need it. I've removed